stacks_queues/queue.py

```diff
@@ -15,47 +15,90 @@ def __init__(self):
         self.front = -1
         self.rear = -1
         self.size = 0
-      
+
 
     def enqueue(self, element):
         """ Adds an element to the Queue
             Raises a QueueFullException if all elements
             In the store are occupied
             returns None
         """
-        pass
+        # Handles the rear_ptr
+        if self.front == -1:
+            self.front = 0
+            self.rear = 1
+            self.store[self.front] = element
+            self.size += 1
+            return self.store
```

Comment:
Your implementation works, but some food for thought: As an alternative implementation, you could set the `rear` pointer to 0 here and your `else` clause at the bottom would actually handle the rest! 
```suggestion
            self.rear = 0
```